rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/NSCreationServlet.java

```diff
@@ -119,16 +127,27 @@ private void createNs(HttpServletRequest request, HttpServletResponse response)
                         readingPolicyParams = true;
                         readingInfraParams = false;
                     } else if (readingInfraParams) {
-                        infraParams.add(fi.getString());
+                        if (fieldName.endsWith(NodeSourceEditWindow.EDIT_OR_UPLOAD_FORM_ITEM_SUFFIX)) {
+                            takeInlineContent = fi.getString().endsWith(NodeSourceEditWindow.EDIT_RADIO_OPTION_NAME);
+                        } else if (fieldName.endsWith(NodeSourceEditWindow.EDIT_FORM_ITEM_SUFFIX)) {
+                            if (takeInlineContent) {
+                                infraFileParams.add(fi.getString());
+                            }
+                        } else {
+                            infraParams.add(fi.getString());
+                        }
                     } else if (readingPolicyParams) {
                         policyParams.add(fi.getString());
                     } else {
                         LOGGER.warn("Unexpected param " + fieldName);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)